hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedSlice.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.hadoop.hdds.utils.db.managed;
+
+import org.rocksdb.Slice;
+
+/**
+ * Managed Options.
+ */
+public class ManagedSlice extends Slice {
+
+  public ManagedSlice(byte[] var1) {
+    super(var1);
+  }
+
+  @Override
+  protected void finalize() throws Throwable {
+    ManagedRocksObjectMetrics.INSTANCE.increaseManagedObject();
```

Comment:
Let's make it an util in `ManagedRocksObjectMetrics`.
```
static void assertClosed(RocksMutableObject rocksObject) {
    ManagedRocksObjectMetrics.INSTANCE.increaseManagedObject();
    if (rocksObject.isOwningHandle()) {
      ManagedRocksObjectMetrics.INSTANCE.increaseLeakObject();
      LOG.warn("{} is not closed properly",
          rocksObject.getClass().getSimpleName());
    }
  }
```

Reply:
@duongnguyen0 , thanks for the code review.  RocksMutableObject.isOwningHandle has a protected access privilege, can cannot be called in either ManagedRocksObjectMetrics or ManagedRocksObjectUtils(I think you assume this class).

Reply:
Sounds good, thanks for the clarification, Sammi.